src/main/java/seedu/address/model/person/task/Task.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.person.task;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.util.ToStringBuilder;
+
+/**
+ * Represents a TaskList's task in the address book.
+ * Guarantees: immutable; is always valid
+ */
+public class Task {
+    public static final String MESSAGE_CONSTRAINTS = "Task should have task description and deadline";
+    public final TaskDescription taskDescription;
+    public final TaskDeadline taskDeadline;
```

Comment:
Should these values be private? Maybe we can implement the compareTo method in Task as well

Reply:
okies making them private makes sense, i'll make sure to also add getter methods so that other parts function properly. adding compareTo makes sense also